ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcLogAppender.java

```diff
@@ -368,6 +368,7 @@ public void onNext(AppendEntriesReplyProto reply) {
       AppendEntriesRequest request = pendingRequests.remove(reply);
       if (request != null) {
         request.stopRequestTimer(); // Update completion time
+        getFollower().updateLastAppendEntriesResponseTime(request.getSendTime()); // Update the last rpc time
       }
```

Comment:
We should updateLastRpcResponseTime() when request == null.
```jav
      if (request != null) {
        request.stopRequestTimer(); // Update completion time
        getFollower().updateLastAppendEntriesResponseTime(request.getSendTime());
      } else {
        getFollower().updateLastRpcResponseTime();
      }
```

Reply:
Done.